README.md

```diff
@@ -1,3 +1,20 @@
-# Map Out Hate
+## Map Out Hate
 
-The purpose of this project is to create a data set for instances of micro-aggresions and discrimination. 
+The purpose of this project is to create a data set for micro-aggressions and discrimination. It is a safe space for people to report incidents that happen to them. It also provides resources for users to take further action and prepare themselves for various situations. When a user reports pins are posted on a map and they are able to visualize all other reports that have come in. Click [here](http://map-out-hate.herokuapp.com/) to explore the application.
```

Comment:
Also, it is generally bad practice to make links that say "click here". Instead, make the link the text "explore the application". This helps with accessibility as well.